what-is-chalk.mdx

```diff
@@ -18,215 +18,467 @@ import { PyDiffEditor } from '@/components/Editor'
 
 <DocsHero />
 
-Chalk is a feature store that enables innovative machine learning teams
-to focus on building the unique products and models that make their
-business stand out. Behind the scenes, Chalk seamlessly handles
-data infrastructure with a best-in-class developer experience.
-Here’s how it works.
+Build, deploy, and iterate faster with Chalk — a programmable feature engine that powers low-latency inference, rapid model iteration, and observability across your model lifecycle.
+Chalk solves core pain points teams face when building enterprise AI and ML systems:
 
----
+- deploying and scaling production-grade infrastructure
+- authoring feature pipelines in pure Python — No DSLs! No rewrites!
```

Comment:
the header for this list of stuff is "core pain points", so this ends up saying "writing code in python sux", kind of the opposite of our intended message. I think either the header needs to change, we should drop the bullets, or some of these bullets need to change. Maybe it would be nice to focus on the positives we enable, instead of what is painful?